blocks/block-edit/index.js

```diff
@@ -20,7 +20,14 @@ function BlockEdit( props ) {
 		Edit = blockType.edit || blockType.save;
 	}
 
-	return applyFilters( 'BlockEdit', <Edit key="edit" { ...editProps } />, props );
+	const filteredElements = applyFilters( 'BlockEdit', { edit: <Edit key="edit" { ...editProps } />, fragments: [] }, props );
```

Comment:
@aduth does it make sense separate the `Edit` element here so that later hooks can rely on it being an element still (in the shape of registered)?  I added a `fragments` key also which works for fills, this could actually be called`fills`...